diff --git a/CMakeLists.txt b/CMakeLists.txt
index 97ee47c..cff5622 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ endif()
 # TODO(henningkayser): Remove policy fix when assimp 5.1 is available
 # Suppress policy warning in assimp (https://github.com/assimp/assimp/pull/2722)
 set(CMAKE_POLICY_DEFAULT_CMP0012 NEW)
-find_package(ASSIMP QUIET)
+find_package(assimp QUIET)
 if(NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
   # assimp is required, so REQUIRE the second attempt
@@ -74,7 +74,7 @@ ament_target_dependencies(${PROJECT_NAME}
   resource_retriever
   console_bridge
   OCTOMAP
-  ASSIMP
+  assimp
   QHULL
 )
 
