diff --git a/CMakeLists.txt b/CMakeLists.txt
index afe0779..bbcb412 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,13 +3,13 @@ cmake_minimum_required(VERSION 3.5)
 project(rcutils)
 
 # Default to C11
-if(NOT CMAKE_C_STANDARD)
-  set(CMAKE_C_STANDARD 11)
-endif()
-# Default to C++14
-if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
-endif()
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
+# if(NOT CMAKE_C_STANDARD)
+# endif()
+# # Default to C++14
+# if(NOT CMAKE_CXX_STANDARD)
+# endif()
 
 include(CheckLibraryExists)
 
diff --git a/include/rcutils/error_handling.h b/include/rcutils/error_handling.h
index 9f19b83..20c0f81 100644
--- a/include/rcutils/error_handling.h
+++ b/include/rcutils/error_handling.h
@@ -88,7 +88,7 @@ typedef struct rcutils_error_state_t
 
 // make sure our math is right...
 #if __STDC_VERSION__ >= 201112L
-static_assert(
+_Static_assert(
   sizeof(rcutils_error_string_t) == (
     RCUTILS_ERROR_STATE_MESSAGE_MAX_LENGTH +
     RCUTILS_ERROR_STATE_FILE_MAX_LENGTH +
diff --git a/src/error_handling.c b/src/error_handling.c
index 868a501..dba424f 100644
--- a/src/error_handling.c
+++ b/src/error_handling.c
@@ -200,7 +200,7 @@ rcutils_set_error_state(
   // Only warn of overwritting if the new error is different from the old ones.
   size_t characters_to_compare = strnlen(error_string, RCUTILS_ERROR_MESSAGE_MAX_LENGTH);
   // assumption is that message length is <= max error string length
-  static_assert(
+  _Static_assert(
     sizeof(gtls_rcutils_error_state.message) <= sizeof(gtls_rcutils_error_string.str),
     "expected error state's max message length to be less than or equal to error string max");
   if (
diff --git a/src/error_handling_helpers.h b/src/error_handling_helpers.h
index a0e898f..e473803 100644
--- a/src/error_handling_helpers.h
+++ b/src/error_handling_helpers.h
@@ -145,7 +145,7 @@ __rcutils_format_error_string(
   static const char format_1[] = ", at ";
   static const char format_2[] = ":";
   char line_number_buffer[21];
-  static_assert(
+  _Static_assert(
     sizeof(error_string->str) == (
       sizeof(error_state->message) +
       sizeof(format_1) - 1 /* minus the null-term */ +
