diff --git a/src/memory_tools/CMakeLists.txt b/src/memory_tools/CMakeLists.txt
index 7cfbba3..3451c95 100644
--- a/src/memory_tools/CMakeLists.txt
+++ b/src/memory_tools/CMakeLists.txt
@@ -34,7 +34,7 @@ if(UNIX AND NOT APPLE)
   # On Linux like systems, add dl and use the normal library and LD_PRELOAD.
 
   if(DL_LIBRARY)
-    target_link_libraries(memory_tools PUBLIC ${DL_LIBRARY})
+    target_link_libraries(memory_tools PUBLIC dl)
   endif()
 endif()
 
diff --git a/src/memory_tools/vendor/bombela/backward-cpp/backward.hpp b/src/memory_tools/vendor/bombela/backward-cpp/backward.hpp
index 6efa46d..546f303 100644
--- a/src/memory_tools/vendor/bombela/backward-cpp/backward.hpp
+++ b/src/memory_tools/vendor/bombela/backward-cpp/backward.hpp
@@ -166,6 +166,11 @@
 //
 // Note that only one of the define should be set to 1 at a time.
 //
+#undef BACKWARD_HAS_DWARF
+#undef BACKWARD_HAS_DW
+#define BACKWARD_HAS_DWARF 0
+#define BACKWARD_HAS_DW 0
+
 #if BACKWARD_HAS_DW == 1
 #elif BACKWARD_HAS_BFD == 1
 #elif BACKWARD_HAS_DWARF == 1
